en_US/messages.json

```
@@ -78,5 +78,35 @@
     "qualityEstimationMessage": {
       "message": "Enable quality estimation",
       "description":"Checkbox to enable the quality estimation of the translation."
+    },
+    "formtranslationsReady": {
+      "message": "Ready",
+      "description":"Notifies that the form translations is ready to be used."
+    },
+    "formtranslationsTyping": {
+      "message": "Typing",
+      "description":"Notifies the user that typing is occuring in the form translations area."
+    },
+    "formtranslationsTranslationInProgress": {
+      "message": "Translation in progress...",
```

```suggestion
      "message": "Translation in progress…",
```

Don't use 3 dots for ellipsis